Dockerfile.in

```diff
@@ -34,7 +34,7 @@ RUN make build
 
 
 #@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cli:v4.7)
-FROM registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cli:v4.7.0-202104250659.p0 AS origincli
+FROM registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cli:v4.8.0-202104250659.p0 AS origincli
```

Comment:
this may not be correct image pull spec
try 
```
registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cli:v4.9.0.20210930.232427
```

Reply:
why v4.`9`.0?